src/main/java/com/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.wolox.training.repositories;
+
+import com.wolox.training.model.Book;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+public interface BookRepository extends CrudRepository<Book, Long> {
+
+    @Query("select b.author from Book b where b.author = :author")
+    Book findBookByAuthor(@Param("author") String author);
```

Comment:
`@Query` is not necesary, the orm takes care of that.
You only need `Optional<Book> findByAuthor(String author)`.
If you want ignore uppercas use `Optional<Book> findByAuthorIgnoreCase(String author)`